tools/hitmap/hitmap.xml

```diff
@@ -0,0 +1,297 @@
+
+<tool id="hitmap" name="HiT-MaP" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" license="MIT">
+    <description>High-resolution Maldi-imaging Proteomics Analysis</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <edam_topics>
```

Comment:
if you like please include a bio.tools identifier. The edam ontology is not strictly needed anymore, we will get those with the bio.tools IDs. See the xref tag.

Reply:
HiT-MaP doesn't have a listing in bio.tools. Should I leave this as edam topics?

Reply:
In this case, leave it! We could also add it to https://github.com/bio-tools/content, but this repo is not yet in production.